### PR TITLE
LPS-143509 When creating a new site shows the categories selectors for the required vocabularies

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-143509

To solve this bug, in the creation of the site we have applied the same logic that is applied in the creation of pages when there are required vocabularies.

Previous PR: https://github.com/liferay-echo/liferay-portal/pull/6801

cc: @Tim-Cao @sandrodw3

### CODE REVIEW

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/add_group.jsp [hunk 1]
```diff
@@ -70,74 +70,12 @@ AddGroupDisplayContext addGroupDisplayContext = new AddGroupDisplayContext(reque
...
-				}
-			});
-	});
-</aui:script>
\ No newline at end of file
+<liferay-frontend:component
+	componentId='<%= liferayPortletResponse.getNamespace() + "addGroup" %>'
+	context='<%=
+		HashMapBuilder.<String, Object>put(
+			"namespace", liferayPortletResponse.getNamespace()
```
Comment (veroglez) [inline modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/add_group.jsp:77]: Here it's not necessary to pass the `namespace`, it already comes by default in the component props.

**Comment by Tim-Cao** [comment]
ci:test:echo

**Comment by Tim-Cao** [comment]
Hi @lfbesada and @sandrodw3, Could you review this issue? The validation style is wrong at the first time.
Actual

**Comment by sandrodw3** [comment]
https://github.com/liferay-echo/liferay-portal/pull/6856

**Pull request closed without merging**